### PR TITLE
Reformat product code

### PR DESCRIPTION
**Pull request by AndriySvyryd**

### CODE REVIEW

**Comment by ajcvickers** [comment]
@AndriySvyryd Any change of style here, or just cleanup with existing style?

**Comment by AndriySvyryd** [comment]
Existing style, but updates to R# might produce slightly different results

**Comment by bricelam** [comment]
I think this broke the build. (ambiguous IsolationLevel)

**Comment by ajcvickers** [comment]
@bricelam @AndriySvyryd Did the PR validation pass?

**Comment by AndriySvyryd** [comment]
@ajcvickers No, we don't yet require that it passes before pushing

**Comment by bricelam** [comment]
We can turn gating on--@davidfowl mentioned he wants to

**Comment by AndriySvyryd** [comment]
Gating on compilation would make sense

**Comment by bricelam** [comment]
@Eilon can you click **Require status checks to pass before merging** for us?

**Comment by smitpatel** [comment]
Gating on testing part should also be fine. Yes, there could be flaky test which may require to be re-run before pull request can be merged but even that should pass in re-run. In worst case, if the validation does not pass, there is always option to merge via command line.

**Comment by bricelam** [comment]
Agree--re-running isn't painful. It will also encourage us to fix flaky tests/infrastructure.

**Comment by Eilon** [comment]
@ajcvickers can do it too. If it's alright, I'd rather have him do it so that as the team wants to change these types of policies Arthur can do it directly. Though, I can certainly do it myself if you want.

**Comment by AndriySvyryd** [comment]
The issue with gating on tests is that they usually take an hour to rerun

File: src/Microsoft.Data.Sqlite.Core/SqliteTransaction.cs [hunk 1]
```diff
@@ -66,8 +65,7 @@ public new virtual SqliteConnection Connection
         protected override DbConnection DbConnection
             => Connection;
 
-        internal bool ExternalRollback
-            => _externalRollback;
+        internal bool ExternalRollback { get; private set; }
```
Comment (bricelam) [inline src/Microsoft.Data.Sqlite.Core/SqliteTransaction.cs]: Not a fan of `private` setters...

File: src/Microsoft.Data.Sqlite.Core/SqliteException.cs [hunk 2]
```diff
@@ -20,14 +20,16 @@ public class SqliteException : DbException
...
+        }
 
         /// <summary>
         ///     Initializes a new instance of the <see cref="SqliteException" /> class.
         /// </summary>
         /// <param name="message">The message to display for the exception. Can be null.</param>
         /// <param name="errorCode">The SQLite error code.</param>
-        /// /// <param name="extendedErrorCode">The extended SQLite error code.</param>
+        /// ///
+        /// <param name="extendedErrorCode">The extended SQLite error code.</param>
```
Comment (bricelam) [inline src/Microsoft.Data.Sqlite.Core/SqliteException.cs]: Well, I suppose it's no more wrong than before 🙂

Comment (bricelam) [reply src/Microsoft.Data.Sqlite.Core/SqliteException.cs]: (Fixed via f7875cf76)

**Comment by ajcvickers** [comment]
Gating turned on. If this becomes problematic, let's discuss. 🌵 🔥 🐐 🤺

**Pull request merged**